doc/en/components/layouts/tile-manager.md

```diff
@@ -0,0 +1,320 @@
+
+```html
+<igc-tile-manager min-column-width="200px" min-row-height="150px">
+  <igc-tile>
+    <span slot="title">Tile 1 header</span>
+    <p>Content for Tile 1</p>
+  </igc-tile>
+  <igc-tile>
+    <span slot="title">Tile 2 header</span>
+    <p>Content for Tile 2</p>
+  </igc-tile>
+  ...
+</igc-tile-manager>
+```
+
+### Example
+
+`sample="/layouts/tile-manager/columngap", height="631", alt="{Platform} Tile Manager Column Example"`
+
+## Tile component
+
+The Tile component has properties that can be set individually for each tile. Some of these properties include:
+
+- The `col-span` property specifies how many columns the tile will span across in the layout, allowing you to control its horizontal size.
+- The `row-span` property determines how many rows the tile will span vertically, adjusting the tile's height within the layout.
+- The `col-start` property specifies the starting column where the tile is placed.
+- The `row-start` property specifies the starting row where the tile is placed.
+- The `disable-resize` property prevents the tile from being resized by the user.
+
+```html
+<igc-tile-manager>
+  <igc-tile col-span="2" disable-resize>
+    <span slot="title">Tile 1 header</span>
+    <p>Content for Tile 1</p>
+  </igc-tile>
+  <igc-tile>
+    <span slot="title">Tile 2 header</span>
+    <p>Content for Tile 2</p>
+  </igc-tile>
+  ...
+</igc-tile-manager>
+```
+
+The Tile component also exposes several slots which you can use:
+
+| Slot name | Description |
+| ---------|------------ |
+| `title` | Content for the tile header. |
+| `fullscreen-action` | Overwrite the default fullscreen action content. |
+| `maximize-action` | Overwrite the default maximize action content. |
+| `actions` | 	Custom content rendered after the default actions. |
+| `side-adorner` | Overwrite the default horizontal resize adorner. |
+| `corner-adorner` | 	Overwrite the default diagonal resize adorner. |
+| `bottom-adorner` | Overwrite the default vertical resize adorner. |
+
+
+### Header section actions
+
+By default, the header section includes two action buttons:
+
+- The `maximize` button enlarges the tile's content to fill the entire width of the Tile Manager, offering a wider view of the content.
+- The `fullscreen` button enables the tile to open in fullscreen mode in the user's browser.
+
+<img src="../../images/tile-manager-actions.png" />
+
+If you want to display just one of the two buttons, you can set it as a slot attribute within the Tile component. Use the `maximize-action` value to show only the maximize button, or the `fullscreen-action` value to show only the fullscreen button.
```

Comment:
The default action buttons could be removed by setting the corresponding properties disableMaximize or  disableFullscreen. The slots are used in case the appearance of the button should be changed.
```suggestion
If you want to display just one of the two buttons, you can set either `disable-maximize` or `disable-fullscreen` attributes. To customize the appearance use the `maximize-action` slot for the maximize button, or the `fullscreen-action` slot for the fullscreen button.
```